clio-client/src/test/scala/org/broadinstitute/clio/client/util/IoUtilSpec.scala

```diff
@@ -10,4 +10,9 @@ class IoUtilSpec extends AsyncFlatSpec with Matchers with TestData {
     IoUtil.readMetadata(metadataFileLocation) should startWith("{")
   }
 
+  it should "Compute the correct md5 hash of a file and return just the hash" in {
+    // This command also works for local files, so we're using that to check the parsing of the output
+    IoUtil.getMd5HashOfGoogleObject(metadataFileLocation) should be("240a1a8639f7142c871a659a85f69cce")
+  }
```

Comment:
Just to sanity-check the stdout parsing, can you add unit tests for ls / du?

Reply:
ls and du don't work on non-cloud paths...

Reply:
Just to TOTB: You could run them on the "current-clio-version.txt" object.